src/__main__.py

```diff
@@ -75,6 +85,7 @@ async def start():
 
 # entry point
 if __name__ == "__main__":
+    prometheus_client.start_http_server(8080, "localhost")
```

Comment:
Are we going to send this out of app for analysis or will this remain?

Reply:
This should probably be at least a cli flag.
and yes, actual analysis occurs externally in prometheus / grafina.
there is very little mecha can do with these objects.